ADAL/src/cache/ADTokenCacheItem+Internal.m

```diff
@@ -255,23 +255,9 @@ - (void)logMessage:(NSString*)message level:(ADAL_LOG_LEVEL)level correlationId:
 {
     if (_tombstone)
     {
-        NSString* tombstoneMessage = nil;
-        if (message)
-        {
-            tombstoneMessage = [NSString stringWithFormat:@"%@ tombstone : %@", message, _tombstone];
-        }
-        else
-        {
-            tombstoneMessage = [NSString stringWithFormat:@"Tombstone : %@", _tombstone];
-        }
+        [ADLogger log:level context:self errorCode:0 correlationId:correlationId userInfo:nil isPii:YES format:@"%@", _tombstone];
+        [ADLogger log:level context:self errorCode:0 correlationId:correlationId userInfo:nil isPii:NO format:@"{\n\tresource: %@\n\tclientId: %@\n\tauthority:%@\n}", _resource, _clientId, _authority];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 201 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)